signac/core/collection_zarr.py

```diff
@@ -16,36 +18,31 @@ class ZarrCollection(SyncedCollection):
 
     backend = __name__  # type: ignore
 
-    def __init__(self, name=None, store=None, **kwargs):
-        import zarr
+    def __init__(self, group=None, **kwargs):
         import numcodecs  # zarr depends on numcodecs
 
-        self._root = zarr.group(store=store)
-        self._name = name
+        self._root = group
         self._object_codec = numcodecs.JSON()
         super().__init__(**kwargs)
-        if (name is None) == (self._parent is None):
-            raise ValueError(
-                "Illegal argument combination, one of the two arguments, "
-                "parent or name must be None, but not both.")
 
     def _load(self):
-        """Load the data from a Radis-database."""
+        """Load the data."""
```

Comment:
I did not notice this before, but the word "data" is typically assumed to be plural in this context:
```suggestion
        """Load data."""
```
Furthermore, I think it would make sense to be more explicit here:
```suggestion
        """Load data from zarr-file."""
```
Please apply everywhere else as well.

Reply:
Minor clarification: Dropping the article “the” is a stylistic choice, not one that depends on data being plural or singular. (Otherwise it’d be a debate of “a data” for singular vs “the data” for plural, not a question of dropping the article.) I am fine with the suggested changes, regardless.

Reply:
I have changed it with `Load the data from zarr-store`.